src/strategies/moloch-all/index.ts

```diff
@@ -0,0 +1,126 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'adridadou';
```

Comment:
This should be your GitHub username
```suggestion
export const author = 'scottrepreneur';
```